inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/project/EvaluationSimulationPanel.java

```diff
@@ -0,0 +1,300 @@
+import de.tudarmstadt.ukp.inception.recommendation.imls.external.ExternalRecommender;
+import de.tudarmstadt.ukp.inception.recommendation.imls.external.ExternalRecommenderFactory;
+import de.tudarmstadt.ukp.inception.recommendation.imls.external.ExternalRecommenderTraits;
+import de.tudarmstadt.ukp.inception.recommendation.imls.opennlp.doccat.OpenNlpDoccatRecommender;
+import de.tudarmstadt.ukp.inception.recommendation.imls.opennlp.doccat.OpenNlpDoccatRecommenderFactory;
+import de.tudarmstadt.ukp.inception.recommendation.imls.opennlp.doccat.OpenNlpDoccatRecommenderTraits;
+import de.tudarmstadt.ukp.inception.recommendation.imls.opennlp.ner.OpenNlpNerRecommender;
+import de.tudarmstadt.ukp.inception.recommendation.imls.opennlp.ner.OpenNlpNerRecommenderFactory;
+import de.tudarmstadt.ukp.inception.recommendation.imls.opennlp.ner.OpenNlpNerRecommenderTraits;
+import de.tudarmstadt.ukp.inception.recommendation.imls.opennlp.pos.OpenNlpPosRecommender;
+import de.tudarmstadt.ukp.inception.recommendation.imls.opennlp.pos.OpenNlpPosRecommenderFactory;
+import de.tudarmstadt.ukp.inception.recommendation.imls.opennlp.pos.OpenNlpPosRecommenderTraits;
+import de.tudarmstadt.ukp.inception.recommendation.imls.stringmatch.StringMatchingRecommender;
+import de.tudarmstadt.ukp.inception.recommendation.imls.stringmatch.StringMatchingRecommenderFactory;
+import de.tudarmstadt.ukp.inception.recommendation.imls.stringmatch.StringMatchingRecommenderTraits;
+
+public class EvaluationSimulationPanel
+    extends Panel
+{
+    private static final long serialVersionUID = 4306746527837380863L;
+
+    private static final String CHART_CONTAINER = "chart-container";
+    private static final String SIMULATION_START_BUTTON = "simulation-start-button";
+    
+    private static final double TRAIN_PERCENTAGE = 0.8;
+    private static final int INCREMENT = 250;
+    private static final int LOW_SAMPLE_THRESHOLD = 10;
+    
+    private static final String FORM = "form";
+
+    private static final Logger log = LoggerFactory.getLogger(EvaluationSimulationPanel.class);
+    
+    private @SpringBean DocumentService documentService;
+    private @SpringBean UserDao userDao;
+    
+    private final WebComponent chartContainer;
+    private final Project project;
+
+    public EvaluationSimulationPanel(String aId, Project aProject,
+            RecommenderEditorPanel aRecommenderEditorPanel)
+    {
+        super(aId);
+        project = aProject;
+
+        Form<Recommender> form = new Form<>(FORM);
+        add(form);
+
+        chartContainer = new Label(CHART_CONTAINER);
+        chartContainer.setOutputMarkupId(true);
+        form.add(chartContainer);
+
+        form.add(new AjaxButton(SIMULATION_START_BUTTON)
+        {
+            private static final long serialVersionUID = -3902555252753037183L;
+
+            @Override
+            protected void onError(AjaxRequestTarget aTarget)
```

Comment:
`LambdaAjaxButton` won't need this error handler. It has it built-in.

Reply:
Done.